pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -318,31 +318,59 @@ public class Utils {
     public static String prepareForEvalToString(String s) {
         String toEval = s ?: ""
         if (!toEval.startsWith('"') || !toEval.endsWith('"')) {
-            toEval = '"' + toEval + '"'
+            if (toEval.indexOf('\n') == -1) {
+                toEval = '"' + toEval + '"';
+            } else {
+                toEval = '"""' + toEval + '"""';
+            }
         }
 
         return toEval
     }
 
+    static List<List<String>> getEnvCredentials(Environment environment, CpsScript script) {
+        List<List<String>> credsTuples = new ArrayList<>()
+        if (environment != null) {
+            credsTuples.addAll(environment?.getCredsMap(script)?.collect { k, v ->
```

Comment:
_nit_ You already know that `environment != null` so you don't need the first `?.`